index.html

```diff
@@ -2967,30 +2867,10 @@ <h1>DID Methods</h1>
             <a href="https://github.com/gatacaid/gataca-did-method">Gataca DID Method</a>
           </td>
         </tr>
-        <tr>
-          <td style="text-decoration:line-through">
-            did:git:
```

Comment:
Opened https://github.com/w3c/did-spec-registries/issues/354 to address this issue.